### PR TITLE
new test that checks for >2 colons in the header

### PR DESCRIPTION
**Pull request by ventura-rivera**
This purpose of this test is to exploit the ValueError raised when a header contains >2 colons.
I created a new function that tests a header with >2 colons.
The try except block expects an error and assigns None to the variable: keyword
The assert statement makes sure None is assigned to keyword

### CODE REVIEW

File: csv_template_ingester/tests/test_template_csv_parser.py [hunk 1]
```diff
@@ -23,6 +23,12 @@ def test_get_keyword():
     assert keyword_two == 'NAME'
     assert column_header_two == ''
 
+def test_get_keyword_colon_count():
+    try:
```
Comment (jling11) [inline csv_template_ingester/tests/test_template_csv_parser.py]: please add code comments for what this test is checking for.

Comment (ventura-rivera) [reply csv_template_ingester/tests/test_template_csv_parser.py]: I added the comments and followed some better practices suggested by Hutch. If you have any more suggestions, let me know!

**Comment by ventura-rivera** [comment]
This is the first time I've used it, and it's a pretty cool little tool. The try statement runs some code. You can declare an error-type in the exception. If that exception happens, the except block is ran (in this case the test passes; I could write pass if that makes it more readable). If the exception does not happen, the else block runs (in this case the test fails and the assert statement runs)

File: csv_template_ingester/tests/test_template_csv_parser.py [hunk 2]
```diff
@@ -23,6 +23,17 @@ def test_get_keyword():
...
 
+def test_get_keyword_colon_count():
+    """
+    Tests the expected ValueError for >2 colons in the header.
+    Test fails if ValueError not thrown
+    """
+    try:
+        keyword, syst, column_header = get_keyword('PROPERTY::: Test name (MPa)')
+    except ValueError:
+        keyword = '>2_colons'
```
Comment (maxhutch) [inline csv_template_ingester/tests/test_template_csv_parser.py]: This `keyword` isn't used later, so you could replace this line with `pass`.

File: csv_template_ingester/tests/test_template_csv_parser.py [hunk 3]
```diff
@@ -23,6 +23,17 @@ def test_get_keyword():
...
+    else:
+        assert keyword == '>2_colons', "ValueError was not raised"
```
Comment (maxhutch) [inline csv_template_ingester/tests/test_template_csv_parser.py]: If the program gets to this line, it has failed the test no matter what.  To make that more clear, you could just `assert False, "ValueError was not raised"`

Comment (jling11) [reply csv_template_ingester/tests/test_template_csv_parser.py]: Yes, that would make a lot more sense to me.

**Comment by ventura-rivera** [comment]
Cool! I made the changes with a pass statement and False assertion. Let me know if I should change anything else

**Pull request merged**